core-bundle/contao/themes/flexible/styles/base/responsive.pcss

```diff
@@ -119,9 +119,9 @@
         display: none;
     }
 
-    .tl_filter strong {
+    /*.tl_filter strong {
         display: none;
-    }
+    }*/
```

Comment:
This leftover can probably be removed?

Reply:
See 796f2bc5b21decfd21620c5dae034940fc88712c